app/assets/stylesheets/modules/_header.scss

```diff
@@ -12,6 +12,11 @@ header {
     font-size: 55px;
     line-height: 1;
     color: #fefeff;
+
+    a, a:hover, a:focus, a:visited {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Each selector in a comma sequence should be on its own single line